src/main/app/src/app/klanten/zoek/bedrijven/bedrijf-zoek.component.ts

```diff
@@ -116,6 +123,20 @@ export class BedrijfZoekComponent implements OnInit {
       plaats: this.plaatsFormField.formControl,
       type: this.typeFormField.formControl,
     });
+
+    this.uuid = crypto.randomUUID(); // Generate a unique form ID
+
+    if (this.syncEnabled) {
+      // Subscribe to select event, ignore own event
+      this.formSelectedSubscription =
+        this.formCommunicationService.formSubmitted$.subscribe(
+          ({ selected, uuid }) => {
+            if (selected && uuid !== this.uuid) {
```

Comment:
Any way to test this logic? Unit tests?

Reply:
Well, very possible but this is such a common pattern in JS/TS; I don't really see the added value. Especially since type safety is covered.